lib/firebase/admin/auth/user_info.rb

```diff
@@ -17,35 +17,35 @@ def initialize(data)
 
         # Gets the ID of this user.
         def uid
-          @data.fetch(:rawId)
+          @data["rawId"]
```

Comment:
What's the purpose of moving away from using `fetch`?

Reply:
Some fields can be `nil`